### ISSUE TITLE
Private sprints show only to the team

### ISSUE THREAD

**Issue by victorfcm** [renatomarinho/laravel-gitscrum#82]
If I login with another user, i'll see every sprint, even if it is private. I think this is not what we want to.. If a sprint is private, it should be visible only to the team that are working on it!

**Comment by victorfcm** [comment]
As you can see in the prints, there are 2 users seeing private sprints..

![captura de tela de 2016-12-20 14-00-26](https://cloud.githubusercontent.com/assets/1637099/21357554/b4ecfe26-c6bc-11e6-821c-bc2d04e7875d.png)
![captura de tela de 2016-12-20 14-00-37](https://cloud.githubusercontent.com/assets/1637099/21357555/b4eee8f8-c6bc-11e6-8903-c9db18b01af2.png)

**Comment by renatomarinho** [comment]
Yes, you are correct. Mode private still not was implemented. This mode should update the repos in provider (github or gitlab). Can you help in this feature?

1. The user can see the private sprint only when has a issue or is owner of the sprint.

What you think? Any other rule?

**Comment by victorfcm** [comment]
My focus is in make GitLab work, but, after that I can do it :)

**Comment by AgusRdz** [comment]
@renatomarinho I created a PR to fix this issue. English and Spanish translations.
